image/postgres/Dockerfile

```diff
@@ -12,6 +12,7 @@ LABEL \
     io.openshift.tags="rhacs,central-db,stackrox" \
     maintainer="Red Hat, Inc." \
     name="rhacs-central-db-rhel8" \
+    # We must set source-location to provide it to our custom Snapshot.
```

Comment:
I don't know how it slipped here but we don't need `source-location` in this non-Konflux `Dockerfile` at all. Let's just delete the comment. A lot of labels here including `source-location` could be deleted too but we could leave this for the person who came up with the idea to copy-paste the file without trying to unify one.
```suggestion
```